WebDriverManager.Tests/FirefoxConfigTests.cs

```diff
@@ -26,7 +26,9 @@ public void DriverDownloadTest()
         [Fact]
         public void GetMatchingBrowserVersionTest()
         {
-            Assert.Throws<NotImplementedException>(GetMatchingBrowserVersion);
+            var version = GetMatchingBrowserVersion();
+            Assert.NotEmpty(version);
+            Version.Parse(version);
```

Comment:
Can you please check here also for a version regex

Reply:
The problem is that `FileVersionInfo.FileVersion` used in Windows removes the last two parts if they are zero. For example yesterday it reported "100.0" for my Firefox installation, but today a new patch was released so it's now "100.0.1". So the regex periodically breaks depending on what's installed.

Reply:
I've changed it to regex check that accept 2 or 3 part version numbers as those seem to be the normal occurrences for Firefox. Hope that's okay.